pycbc/conversions.py

```diff
@@ -912,10 +892,13 @@ def get_lm_f0tau_allmodes(mass, spin, modes):
     spin : float or array
         Dimensionless spin of the final black hole.
     modes : list of str
-        The modes to get. Each string in the list should be formatted 'lmN',
-        where l (m) is the l (m) index of the harmonic and N is the number of
-        overtones to generate (note, N is not the index of the overtone). For
-        example, '221' will generate the 0th overtone of the l = m = 2 mode.
+        The modes to get. Each string in the list should be formatted
+        'lmN[nm]', where l (m) is the l (m) index of the harmonic and N is the
+        number of overtones to generate (note, N is not the index of the
+        overtone). If a mode has the string "nm" added to the end of it (for
+        negative m), the -m mode will be used. For example, '221' will generate
```

Comment:
Do I understand correctly that, if I ask for `221nm` I will only get the (2,-2,0) mode? So if I want both positive and negative modes, I need to request both `221` and `221nm`?
If that isn't elsewhere in the code already (I am having a hard time following all the changes), could we also have an option that if one requests `221{x}` then one would get *both* the +m and the -m mode (with `x` any name of your choice)?

Reply:
Removed.